### PR TITLE
Add player icon

### LINKED ISSUE #63: Story: Prototype username displays in-game
Steam allows up to 32 characters, which can be a lot on screen.

One example, perhaps could be toggleable:

image

Perhaps if someone hits tab, we could show this:
image

### LINKED ISSUE #83: Find & Destroy the Little White Circle
image

I introduced this into the StoryMenu at some point, but much to my bewilderment I am struggling to discover where it is an what it is.

### PR DESCRIPTION
**Pull request by 6fears7**
- Pain itself
- ~~Adds the JollyMeter (flat sprites showing # of players in-game + if alive / dead)~~
- Technically adds the 'missing slugcat' icon when at gates or sleeping
- Adds the PlayArrow + Username
- Adds Remix Options Menu for Rain Meadow
- Adjusts some StoryMenu display

## Pain Points:
- The player arrows logic currently takes the init single player hud and iterates over all players and gives them the arrow. For Player 1, this is a problem because they don't have anyone else in-game at the time it's assigned. 
For Player 2, this works except that Player 1's name is now Player 2's name and Player 2's name is Player 1's name. 

Pictured:

image

- This might be partially due to when I'm checking the index position of two different lists, OnlineManager.players always puts you at the top of the list, which should not be occurring. The list should append any newer results to the end of the list. 
- I'm currently trying to use the Update() function for the arrow to help Player 1 check if new people have joined the game and if Player 1 needs to add player arrows to those players. 

## TODOs:
- JollyMeter does not respect individual player colors. Each client dictates their body color for all slugcat icons
- Same goes for the arrow / username
- OnlinePlayerSpecificHUD is a wildly large file and should be split up to make it easier to address the sub components (death, missing slug cat, arrow)

closes #63 
closes #83

### CODE REVIEW

**Comment by 6fears7** [comment]
Ok, found the section that lets me update when other players join. My most recent commit now adds the username to the second player.

Player 1 joins, gets an arrow
Player 2 joins, gets arrows for both itself and Player 1
Player 1 then hands its arrow over to Player 2
Player 2 sees its correct name and Player 1 correct name
At this point, I'm convinced that OnlineManager.players.isMe is being shunted to the front of the list which is causing some of the issues.

**Comment by 6fears7** [comment]
image

Believe I fixed usernames

File: Story/RainMeadow.StoryHooks.cs [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
 using System.Collections.Generic;
-using MonoMod.Cil;
-using System;
-using Mono.Cecil.Cil;
+using JollyCoop.JollyHUD;
 using HUD;
-using System.Text.RegularExpressions;
-
+using System.Linq;
+using System.Security.Cryptography;
```
Comment (Chrometurtle) [inline Story/RainMeadow.StoryHooks.cs:8]: This needed?

Comment (6fears7) [reply Story/RainMeadow.StoryHooks.cs:8]: Nope, I will go back through and remove all these auto-added libraries or just wait for @henpemaz to complete his rewrite for the hud

**Review by Chrometurtle — CHANGES_REQUESTED** [review]
henpe rework requested

**Comment by 6fears7** [comment]
Tested, looks good. Thanks again for the rewrites, merge when you're ready.

**Pull request merged**